util/omrutil/j9memclr.c

```diff
@@ -129,7 +139,7 @@ OMRZeroMemory(void *ptr, uintptr_t length)
 		}
 	}
 
-	if (1 == isZ10orGreater) {
+	if (1 == pfd_to_zero_object_heap_memory && 1 == isZ10orGreater) {
```

Comment:
Same comment as the above if-statement:
```suggestion
	if (usePFD && isZ10orGreater) {
```

Maybe we should change the type of `isZ10orGreater` from `int` to `BOOLEAN`.

Reply:
updated!